contracts/GameJamProxy.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity 0.5.12;
+
+import "openzeppelin-solidity/contracts/access/Roles.sol";
+import "./interfaces/IRegistry.sol";
+import "./GameJamCommon.sol";
+
+
+contract GameJamProxy is GameJamCommon {
```

Comment:
Is it needed for GameJamProxy to inherit from GameJamCommon if we are doing delegated calls ?

Reply:
For some reason (tbh I'm not 100% sure why) we need both the the GameJam Contract and the Proxy contract to inherit from a Common data contract, otherwise none of the modifiers or data storage will work when calling data through the Proxy contract. 

Side note:
> The GameJamProxy just stores data and the execution happen on the GameJam contract